scripts/kubectl-kubestellar-deploy

```diff
@@ -20,7 +20,10 @@
 
 KSDIR=$(cd $(dirname ${BASH_SOURCE[0]}); cd ..; pwd)
 
-image_tag=b23-09-15-04-52-13
+image_tag=$(kubestellar-release 2> /dev/null)
```

Comment:
This will not work because `kubestellar-release` is a special thing that only exists in release archives.  We need developers to be able to build and deploy whatever version they are working on.  I suspect the motivation here is the problem you reported with using recently built images. I think the right fix is the other one that you suggested, which I have put into #1047 .